plugins/modules/modprobe.py

```diff
@@ -46,6 +46,7 @@
         type: str
         choices: [ disabled, absent, present ]
         default: disabled
+        version_added: '7.0.0'
```

Comment:
We usually only use quotes when necessary. Since this isn't a number, it's enough to simply provide the version string without quotes:
```suggestion
        version_added: 7.0.0
```